docs/configuration.md

```diff
@@ -146,37 +146,38 @@ See [module options](#module-options) for installing custom databases.
 
 Configure the welcome message.
 
-If set to true then a welcome message is printed in the log at startup. It defaults to true.
+If set to true, then a welcome message is printed in the log at startup. It defaults to true.
 
 ```yaml
 welcome-message: true
 ```
 
 ### Logging
 
-Configure logging in opsdroid.
+Configure logging in opsdroid. If you don't have logging settings configured, opsdroid will use rich logging.
```

Comment:
> If you don't have logging settings configured

Which settings are "logging settings"? Anything under `logging.*`? For example, will configuring a `logging.path` disable `rich` logging? What about the `whitelist` or `blacklist`? Or are you only talking about `logging.formatter` (which you've documented below)? Will configuring `logging.formatter` disable `rich` or will the setting be ignored?

Reply:
If you install opsdroid, the logging in the configuration file is commented out. If you don't specify any logging you get rich logging.

Similarly before the PR if you don't specify any logging you get the console log.